### PR TITLE
Build cleanup

### PR DESCRIPTION
**Pull request by danbev**

### CODE REVIEW

**Comment by danbev** [comment]
@lholmquist Do you have time to take a look at this? 
It's just a start and I'll go through the additional test and the code which most probably will be refactored/rewritten. But I'd like to get all the infrastructure around this right before starting.

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "semistandard",
+  "env": {
```
Comment (lholmquist) [inline .eslintrc.json]: probably can get rid of this `env` since we aren't going to use mocha

**Comment by danbev** [comment]
Ah good catch! Thanks

måndag 7 november 2016 skrev Lucas Holmquist <notifications@github.com>:

**Comment by danbev** [comment]
@corinnekrych I'd like to merge this work as I think it is in good enough shape to continue later at some stage. I've basically tried to re-enable all the test and also follow the recommendations provided by [bucharest-gold/entente](https://github.com/bucharest-gold/entente).

**Comment by corinnekrych** [comment]
👁 👁

**Comment by corinnekrych** [comment]
@danbev I wonder if we don't miss one step in travis.yml to run the instrumentation itself for coverall, see:
http://codeheaven.io/javascript-code-coverage-with-instanbul-and-coveralls/#code-coverage-tool
like:
babel-node ./node_modules/istanbul/lib/cli cover _mocha test/index.js

**Comment by danbev** [comment]
@corinnekrych Ah yes, the coverage run script is not included in the `ci` run script. Let me add it and that should do the trick. Thanks!

**Comment by corinnekrych** [comment]
LGTM
Nice to show a good coverage :)

**Comment by danbev** [comment]
@corinnekrych Thanks. I've rebased this and squashed the commits, also update the project version and I'll then merge it once CI passes.

**Pull request merged**